template/src/components/DebugMenu.js

```diff
@@ -0,0 +1,259 @@
+import BottomSheet, {BottomSheetFlatList} from '@gorhom/bottom-sheet'
+import React, {useCallback, useMemo, useRef, useState} from 'react'
+import {
+  Modal,
+  Pressable,
+  SafeAreaView,
+  ScrollView,
+  View,
+  Text,
+  Button,
+  StyleSheet,
+  useWindowDimensions,
+} from 'react-native'
+import {getApplicationName, getBuildNumber, getDeviceId} from 'react-native-device-info'
+import Draggable from 'react-native-draggable'
+
+import Config, {BOTTOM_SHEET_TYPE, CODEPUSH_KEYS, EXTRA_QA_ENVS} from '../constants/configs'
+
+import {InfoMenu, InfoMenuRow, InfoMenuLink} from './InfoMenu'
+
+import {colors, fonts, metrics} from '../themes'
+
+import {localize} from '../locale/I18nConfig'
+import {useDispatch, useSelector} from 'react-redux'
+import {getApiUrl, getCodePushKey} from '../store/selectors'
+import {appActions} from '../store/reducers'
+
+const debugMenuSize = 50
```

Comment:
I think we can move this to file `constants`

Reply:
only this file is used, so dont need to move to constant
const DEBUGMENU_SIZE = 50

Reply:
Okay